independent-publisher-connectors/IBM Watson Text to Speech/apiDefinition.swagger.json

```diff
@@ -0,0 +1,311 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "IBM Watson Text to Speech",
+    "description": "The IBM Watson Text to Speech service provides APIs that use IBM's speech-synthesis capabilities to synthesize text into natural-sounding speech in a variety of languages, dialects, and voices.",
```

Comment:
@lucastitus Please remove the word "API" form the description. Please follow our requirements [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector) and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).